tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml

```diff
@@ -72,6 +72,11 @@ jobs:
         arguments: '$(BuildCommand) --update --config RelWithDebInfo'
         workingDirectory: '$(Build.BinariesDirectory)'
 
+    - ${{ if notIn(parameters['sln_platform'], 'Win32', 'x64') }}:
+      # Use cross-compiled protoc
+      - script: |
+         @echo ##vso[task.setvariable variable=ProtocDirectory]$(Build.BinariesDirectory)\host_protoc\Release
```

Comment:
Why do you need to do this?

Reply:
It's built here: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/build.py#L783

When cross-compiling, protoc must be in the host architecture. It's used as part of the build system to compile the protobuf files.

I reference it here: https://github.com/microsoft/onnxruntime/pull/2987/files#diff-2a44500e30cc605574f7ac04ba287752R10